spyder/widgets/calltip.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, parent=None, as_tooltip=False):
         """
         Shows tooltips that can be styled with the different themes.
         """
-        super(ToolTipWidget, self).__init__(parent, Qt.ToolTip)
+        super(ToolTipWidget, self).__init__(None, Qt.ToolTip)
```

Comment:
Why does the parent of this widget need to be None?

@goanpeca, could this have other undesired side effects?

Reply:
@ccordoba12 please see the comment above